plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -406,20 +406,23 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
 
       incoming_transaction_queue _pending_incoming_transactions;
 
-      void on_incoming_transaction_async(const transaction_metadata_ptr& trx, bool persist_until_expired, next_function<transaction_trace_ptr> next) {
+      void on_incoming_transaction_async(const packed_transaction_ptr& trx, bool persist_until_expired, next_function<transaction_trace_ptr> next) {
          chain::controller& chain = chain_plug->chain();
          const auto max_trx_time_ms = _max_transaction_time_ms.load();
          fc::microseconds max_trx_cpu_usage = max_trx_time_ms < 0 ? fc::microseconds::maximum() : fc::milliseconds( max_trx_time_ms );
 
-         auto after_sig_recovery =
-               [self = this, trx, persist_until_expired, next{std::move(next)}]() mutable {
-                  app().post(priority::low, [self, trx{std::move(trx)}, persist_until_expired, next{std::move(next)}]() {
-                     self->process_incoming_transaction_async( trx, persist_until_expired, next );
```

Comment:
I don't know whether this matters, but next is being copied here.

Reply:
Yes it was. I tested a version where the lambda was mutable and it `std::move` but didn't make any difference.

https://github.com/EOSIO/eos/pull/7721/commits/b734e56e41b2875eb48e491feef6af3a6e8ea9e5